library/Model.re

```diff
@@ -8,6 +8,24 @@ module Make = (Config: Config) => {
   open Ezpostgresql.Pool;
   open Lwt_result.Infix;
 
+  let create = (~connection, ~fields, ~values, ()) => {
```

Comment:
```suggestion
  let insert = (~connection, ~fields, ~values, ()) => {
```

I think we should use the SQL name

Reply:
fields values work but can be a bit error prone, let's do a `(~connection, fields)`? where `fields: list((string, string))`  so `(("name", "Fulano"), ("age", "24"))`